_modules/zoomdata.py

```diff
@@ -143,10 +143,13 @@ def list_repos(compact=False):
         try:
             if repo_root == 'latest':
                 repo_config['release'] = repo_root
-            # Float release number is more than None and
-            # less than ``latest`` string
-            elif float(repo_root) > repo_config['release']:
-                repo_config['release'] = float(repo_root)
+            else:
+                # repo_root is a string like '5.8'
+                if isinstance(repo_config['release'], type(None)):
+                    repo_config['release'] = float(repo_root)
+                elif isinstance(repo_config['release'], float) and \
+                        float(repo_root) > repo_config['release']:
+                    repo_config['release'] = float(repo_root)
```

Comment:
Cool stuff! :+1:

Reply:
The formula was tested to work with SaltStack 2019.2.5-py3